experimental/uefi/runtime/src/lib.rs

```diff
@@ -38,9 +38,10 @@ pub struct SerializeableRequest {
     pub session_id: SessionId,
     pub body: Vec<u8>,
 }
+pub type SerializedRequest = Vec<u8>;
```

Comment:
IMO this makes things less readable (since we are really only talking about bytes), but I imagine it will get better again with the framing. But in that case, how will this change? Are you thinking of changing this type to something else then?